package.json

```diff
@@ -150,9 +150,9 @@
   "type": "module",
   "exports": {
     ".": {
+      "types": "./types/index.d.ts",
       "import": "./src/index.js",
```

Comment:
The `"types"` field must be declared first, because TypeScript is using first matching condition. For instance, when `"imports"` is declared first and library is imported in an ESM project, TypeScript will look for type definitions in `./src/index.d.ts`. But that does not exist.

Try changing the order and you should see an error in your editor. (If not, run TSTyche and it will catch this problem.)